config/formats.js

```diff
@@ -65,7 +65,7 @@ exports.Formats = [
 	{
 		name: "[Gen 7] UU",
 		desc: [
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3612054/\">UU Metagame Discussion</a>",
+			"&bullet; <a href=\"http://www.smogon.com/forums/threads/3614035/"\">UU Metagame Discussion</a>",
```

Comment:
You have an extra `"` which is causing the crash.

Although, really, these should probably all be converted to `` ` `` quotes anyway.